app/models/project_step.rb

```diff
@@ -385,12 +398,10 @@ def handle_finalized_at
   end
 
   def handle_schedule_children
-    return unless persisted? && schedule_children.present? &&
-      (scheduled_start_date_changed? || scheduled_duration_days_changed?)
-
-    schedule_children.each do |step|
-      step.scheduled_start_date = scheduled_end_date
-      step.save
+    # return unless persisted? && schedule_children.present? &&
```

Comment:
Let's discuss this. The steps were not updating correctly and many steps are likely out of sync with their precedent steps. Perhaps a migration would be the better way to resolve these items being out of sync.

But moving forward, the schedule children should be updating any time the duration, scheduled date, or completed date changes on a precedent step.